streams/src/test/java/org/apache/kafka/streams/kstream/internals/graph/GraphGraceSearchUtilTest.java

```diff
@@ -60,8 +62,7 @@ public void close() {}
                 },
                 "dummy"
             ),
-            null,
-            null,
+            (StoreBuilder<? extends StateStore>) null,
```

Comment:
The compiler can't tell which constructor to call, so we cast null to pick one. It doesn't matter which, since this argument is never evaluated.